index.html

```diff
@@ -84,10 +91,26 @@ <h2 class="center mg-1"> Main Program </h2>
                 <p>Listen to the speakers from various countries about the messages of sharing and opening.</p>
             </div>
         </div>
-        <button class=" mg-1 bg-primary text-white btn btn-main center" type="button"> Join CC global Summit 2015 </button>
+        <button class=" mg-1 bg-primary text-white btn btn-main btn-main-desktop center" type="button"> See the whole Program </button>
+
+        <button class=" mg-1 bg-primary text-white btn btn-main btn-main-mobile center" type="button"> Join CC global Summit 2015 </button>
     </main>
     <section class="container grid flow ff-lato drivers">
-        <h2 class="center drivers-header">Drivers</h2>
+        <h2 class="center drivers-header mg-2">Drivers</h2>
```

Comment:
- [ ] [OPTIONAL] This section should be generated dynamically, please use JavaScript to generate this HTML code. when creating the JavaScript code kindy consider:

    - The list of speakers used to generate the HTML is assigned to a variable as an array or object and holds their specific data 
    e.g.
   ```
   {
  name: "Julia Leda", 
  role: "President of Young pirates of Europe", 
  description: "European ingetration, political democracy, and participation of ...", 
  photo: "image_url_here"
  }
   ```
   - There is a function that iterates through the speaker's list on window load (DOM load) and creates HTML elements with the data from each object
   - The dynamic content is attached to the static HTML by selecting the target parent element and adding the generated HTML with the DOM's `appendChild()` or `innerHTML()`